CDTDatastore/Encryption/CDTEncryptionKey.h

```diff
@@ -27,14 +27,15 @@
 // Size (bytes) of the DPK (Data Protection Key)
 #define CDTENCRYPTIONKEY_KEYSIZE 32
 
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
I'm confused what value this is providing here, with this in place shouldn't we remove:
* the `nonnull` from the attribute list on line 36
* the `nonnull` annotations from `NSData *` on lines 47 and 51
* the `nonnull` from `CDTEncryptionKey *` on line 49